package-lock.json

```diff
@@ -2133,12 +2133,6 @@
           "dev": true,
           "optional": true
         },
-        "ini": {
-          "version": "1.3.5",
-          "bundled": true,
-          "dev": true,
-          "optional": true
-        },
```

Comment:
this is under 'fsevents'

Reply:
https://github.com/fsevents/fsevents
"The FSEvents API in MacOS allows applications to register for notifications of changes to a given directory tree. It is a very fast and lightweight alternative to kqueue."